Karamelfile

```diff
@@ -3,6 +3,12 @@ dependencies:
     global:  
       - ndb::mysqld
       - hops::docker_registry
+      - hops::nm
+      - hops::dn
+  - recipe: hops_airflow::install
+    global:
+      - hops::nm
+      - hops::dn
```

Comment:
Karamel doesn't support dependencies for the install recipes so this can be removed.

Reply:
But when I remove the dependancy, the install recipe was being run before the namenode and was failing. I will retest.